doc/manuscript.Rmd

```diff
@@ -131,20 +127,19 @@ knitr::include_graphics(here::here(purdy_pictures_by_age$predictor_pa_volume), e
 
 We estimated how physical activity intensity affects sleep across different age groups
 We present the results controlling for sex, SES, and BMI, in Table \@ref(tab:sleep-outcomes) and Figure \@ref(fig:sleep-by-intensity-fig).
-We found that higher physical activity intensity is directly proportional to longer sleep duration and better sleep efficiency.
-In the case of older participants, physical activity intensity had a U-shaped relationship with sleep onset, meaning that individuals with very low or very high physical activity intensity had longer sleep onset.
-We also found a strong link between physical activity intensity and improved sleep regularity, which weakened at higher intensity levels.
+We found no evidence that physical activity intensity is related to sleep duration, sleep efficiency, or sleep onset.
+However, we did find evidence for an association between physical activity intensity and improved sleep regularity, which weakened at higher intensity levels.
```

Comment:
line 131: This looks like it's weakening as a function of age, rather than intensity level. We don't have evidence for the poly terms.

![image](https://github.com/Motivation-and-Behaviour/sleepIPD_analysis/assets/12742211/baa84847-a8f4-4a08-b186-f9be542d6d15)

![image](https://github.com/Motivation-and-Behaviour/sleepIPD_analysis/assets/12742211/3fdcd36c-3d48-44a3-b441-68aab78dd0d2)